index.js

```diff
@@ -0,0 +1,123 @@
+"use strict";
+
+class WebError extends Error {
```

Comment:
A simple error class that bakes in the idea of specifying a status.

I used a subclass of Error to avoid an extra, distracting entry in all the stack traces reported, as we had before when I used an `error` method of minuscule instead.